lib/util/git/auth.spec.ts

```diff
@@ -191,6 +191,26 @@ describe('util/git/auth', () => {
       });
     });
 
+    it('returns url with token containing username for GitLab token without hostType', () => {
+      expect(
+        getGitAuthenticatedEnvironmentVariables('https://gitlab.com/', {
+          token: 'token1234',
+          matchHost: 'github.com',
```

Comment:
```suggestion
          matchHost: 'gitlab.com',
```

Reply:
right, it's only used for logging, so that's why nothing failed 😅 

https://github.com/renovatebot/renovate/blob/fb382c25f9ce284ae664d1abc2579aa2b2f1aca5/lib/util/git/auth.ts#L18-L25